.github/workflows/docs-test.yaml

```diff
@@ -0,0 +1,29 @@
+name: DITA test
+
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+    paths: ['documentation/**/*']
+
+jobs:
+
+  docs:
+    name: Docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+          bundler-cache: true
+      - name: Update gems
+        run: bundle update
+      - name: Generate docs
+        run: bundle exec rake docs:all
+      - name: Run DITA
+        uses: dita-ot/dita-ot-action@master
+        with:
+          build: |
+            dita -i documentation/puppetdb.ditamap -o out -f html5 --processing-mode=strict
```

Comment:
We should just add this bit to the existing docs workflow (https://github.com/puppetlabs/bolt/blob/main/.github/workflows/docs.yaml).

Reply:
I did originally, but the `paths-ignore: ['**.md', 'schemas/*']` confused me. Does that only ignore `.md` files in the root directory?

Reply:
No, that'll ignore all markdown files in the repo. Since we want to make sure the dita build succeeds, I think removing `**.md` here is fine.